scripts/fix-markdown-violations.sh

```
@@ -0,0 +1,295 @@
+# Function to add language specifiers to code blocks (MD040)
+fix_code_block_languages() {
+    echo -e "${YELLOW}Fixing code block language specifiers (MD040)...${NC}"
+    
+    find "$REPO_ROOT" -name "*.md" -type f \
+        ! -path "*/node_modules/*" \
+        ! -path "*/.git/*" \
+        ! -path "*/system-configs/.claude/agents/*.md" \
+        -print0 | while IFS= read -r -d '' file; do
+        
+        # Look for code blocks without language specifiers
+        if grep -q '^```$' "$file"; then
```

Shellcheck (0.10.0)

[warning] Did you forget to close this double quoted string?

(SC1078)